packages/dashboard/src/msw/iot-sitewise/resources/assetModels/index.ts

```diff
@@ -8,6 +9,27 @@ const reactorAssetModelMinTemperatureId = uuid();
 const reactorAssetModelPressureId = uuid();
 const reactorAssetModelMaxPressureId = uuid();
 const reactorAssetModelMinPressureId = uuid();
+const compositeModelId = uuid();
+const compositeAttributeId = uuid();
+
+const assetModelCompositeModels: AssetModelCompositeModel[] = [
+  {
+    id: compositeModelId,
+    name: 'site-composite',
+    type: 'composite-model',
+    properties: [
+      {
+        name: 'status (composite)',
```

Comment:
Preferably, try and be creative and make these composite model properties sound like real things which would be in a factory.